src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -113,7 +118,10 @@ private static string[] GetDefaultLetterKeyALL(LetterKey letter)
                 .Union(GetDefaultLetterKeySR(letter))
                 .Union(GetDefaultLetterKeySV(letter))
                 .Union(GetDefaultLetterKeyTK(letter))
```

Comment:
not part of this PR. should we add .OrderBy() here, to have letters sorted? That way similar letters would end up next to each other and not ordered by lang

Reply:
Users have a order by use frequency option so they get the ones they use the most first :)